components/teams-service/server/v2/server.go

```diff
@@ -3,9 +3,11 @@ package v2
 import (
```

Comment:
Do we want to keep the `v2` in this package's name? I suppose we could now move the code to `server/server.go`?

Reply:
Ah, looks like we're tracking this in #3060.